KMB/kmb_massload.py

```diff
@@ -161,7 +161,7 @@ def process_date(entry):
 
 def process_byline(entry):
     """Handle unknown entries and rearrange names."""
-    if entry['byline'] in ('Okänd, Okänd', 'Okänd'):
+    if entry['byline'] in ('Okänd, Okänd', 'Okänd', 'Okänd Fotograf'):
```

Comment:
Are we sure that these are all possible variations? Maybe use .lower() just to be able to handle "okänd fotograf" etc. if it crops up? Alternatively, since _okänd_ is the common part here, use `if "okänd" in entry['byline'].lower()`, unless there's some weird cases where this word is used in a different way.